spinegeneric/cli/generate_figure.py

```diff
@@ -152,6 +157,19 @@
     't1': 'T1 [ms]',
     }
 
+# fetch metric field for Plotly
+metric_to_label_plotly = {
```

Comment:
duplication with `metric_to_label`

Reply:
I had to make `metric_to_label_plotly` since Plotly does not understand Latex. If I use `metric_to_label`, y label would look like this for example. 
<img width="91" alt="Capture d’écran, le 2020-11-25 à 15 20 24" src="https://user-images.githubusercontent.com/71463883/100277875-e6fa5300-2f31-11eb-9a46-aae932c18c0d.png">

Reply:
and does `mm<sup>2</sup>` work with matplotlib?

Reply:
It does not, here is an example :
<img width="74" alt="Capture d’écran, le 2020-11-25 à 16 18 00" src="https://user-images.githubusercontent.com/71463883/100282602-ea91d800-2f39-11eb-9a57-52b49b49f0a2.png">

Reply:
ok, in that case you can ignore my comment 😊

Reply:
but please: add a comment explaining why you need to create this new variable (every code that is not obvious should be documented)